lib/core/seleniumRunner.js

```diff
@@ -62,26 +62,29 @@ class SeleniumRunner {
         });
       })
       .tap(() => {
-        let viewPort = this.options.viewPort;
-        if (viewPort) {
-          if (viewPort !== 'maximize') {
-            viewPort = viewPort.split('x');
-          }
-          const window = this.driver.manage().window();
+        let isChromeRunningOnMobile = this.options.mobile;
```

Comment:
Sorry the code hard to read. The right cli option is **chrome.android.package** then we know we are on Android (you cannot run on Android without or ... do you have special hack, maybe it is possible?). You can use lodash.get to get it (we are inconsistent in the code, but that is the right way 2 do it).